src/Export/Export.tsx

```diff
@@ -0,0 +1,87 @@
+import { Show, createSignal, onMount } from "solid-js";
+export function Export() {
+
+function  onExport(event:Event){
+event.preventDefault();
+  }
+
+  return (
+    <>
+      {/* @ts-ignore */}
+      <button onclick="ExportModal.showModal()"
+        class="btn btn-xs btn-neutral absolute top-1 right-5 rounded-md "
```

Comment:
```suggestion
        class="text-sm absolute top-1 right-5"
```

I prefer to have a simple text instead of a button as shown in the issue https://github.com/madil4/awatif/issues/258